### DISCUSSION TITLE
Entity Hider Disable In Wilderness

### DISCUSSION THREAD

**Discussion by PJGJ210** [runelite/runelite#16280] [Ideas]
While I use player indicators (shows name/square even with hider on) I also tend to have entity hider enabled. I would think a toggle to disable while in wildy/pk areas (or specifically for player models?) would be useful. 

Seems similar to the presets thing but automatic disable when going into wildy to not miss a pker...

**Comment by Jbleezy** [comment]
The next update for Dynamic Entity Hider from the Plugin Hub will add this.

https://github.com/runelite/plugin-hub/pull/3904
